### PR TITLE
Update highlander.txt

### PR DESCRIPTION
**Pull request by Spiderbyte12**
Foundations points update

https://7ph.com.au/points-update/points-announcement-foundations/

Fable of the Mirror-Breaker (+1 to 1)
Psychic Frog (+1 to 1)
White Plume Adventurer (+1 to 3)
Phlage, Titan of Fire’s Fury (+1 to 1)
Gush (+1 to 2)
Tolarian Academy (+1 to 2)

### CODE REVIEW

File: js/cards/highlander.txt [hunk 1]
```diff
@@ -54,7 +56,9 @@ Nadu, Winged Wisdom 1
 Natural Order 1
 Oath of Druids 1
 Orcish Bowmasters 1
+Phlage, Titan of Fire’s Fury 1
```
Comment (Fryyyyy) [inline js/cards/highlander.txt]: `Phlage, Titan of Fire's Fury 1`

**Review by Fryyyyy — CHANGES_REQUESTED** [review]
Plz fix the quote mark

File: js/cards/highlander.txt (hunk 1 above)
Comment (Spiderbyte12) [reply js/cards/highlander.txt]: That's what I get for copying card names from somewhere other than scryfall/gatherer

**Comment by Fryyyyy** [comment]
Thank you for doing this!

**Pull request merged**